extensions/insights-default/package.json

```diff
@@ -6,7 +6,7 @@
         "vscode": "*"
     },
     "contributes": {
-        "insights": [
```

Comment:
We might want to add a back compat fix for this before we merge into master. @kburtram are we responsible to maintaining back compat for this kind of stuff